lib/events/firestoreevents/firestoreevents.go

```diff
@@ -457,93 +440,83 @@ func (l *Log) searchEventsOnce(fromUTC, toUTC time.Time, namespace string, limit
 	case types.EventOrderDescending:
 		firestoreOrdering = firestore.Desc
 	default:
-		return nil, 0, "", trace.BadParameter("invalid event order: %v", order)
+		return nil, "", trace.BadParameter("invalid event order: %v", order)
 	}
 
 	query := modifyquery(l.svc.Collection(l.CollectionName).
 		Where(eventNamespaceDocProperty, "==", apidefaults.Namespace).
 		Where(createdAtDocProperty, ">=", fromUTC.Unix()).
 		Where(createdAtDocProperty, "<=", toUTC.Unix()).
 		OrderBy(createdAtDocProperty, firestoreOrdering)).
+		OrderBy(firestore.DocumentID, firestore.Asc).
```

Comment:
As far as I can tell from my limited testing and past knowledge of Firestore, adding this `OrderBy` will require us to add an additional index to `ensureIndexes()`. This will not be caught by tests, since in my experience, the emulator which we use for integration tests does not support indexes.